wpimath/src/main/java/edu/wpi/first/math/MatBuilder.java

```diff
@@ -39,34 +38,4 @@ public static final <R extends Num, C extends Num> Matrix<R, C> fill(
     }
     return new Matrix<>(new SimpleMatrix(rows.getNum(), cols.getNum(), true, data));
   }
-
-  /** Number of rows. */
-  protected final Nat<R> m_rows;
-
-  /** Number of columns. */
-  protected final Nat<C> m_cols;
-
-  /**
-   * Creates a new {@link MatBuilder} with the given dimensions.
-   *
-   * @param rows The number of rows of the matrix.
-   * @param cols The number of columns of the matrix.
-   * @deprecated Use {@link MatBuilder#fill} instead.
-   */
-  @Deprecated(since = "2024", forRemoval = true)
-  public MatBuilder(Nat<R> rows, Nat<C> cols) {
-    this.m_rows = Objects.requireNonNull(rows);
-    this.m_cols = Objects.requireNonNull(cols);
-  }
-
-  /**
-   * Fills the matrix with the given data, encoded in row major form. (The matrix is filled row by
-   * row, left to right with the given data).
-   *
-   * @param data The data to fill the matrix with.
-   * @return The constructed matrix.
-   */
-  public final Matrix<R, C> fill(double... data) {
```

Comment:
This function wasn't deprecated.  Why was it removed?

Reply:
Probably because `MatBuilder` instances (and by extension instance methods) are deprecated?

Reply:
Deprecated, but not removed.  There can be existing code that uses this function.

Reply:
This function can't be deprecated without breakage because it relies on class-level generics. We could use function-level generics, but that's a user-facing breaking change, which defeats the purpose of deprecating it.